.circleci/config.yml

```diff
@@ -51,6 +51,28 @@ jobs:
           name: Clean up reusable Docker
           command: clean_up_reusable_docker
 
+  push-to-dockerhub:
+    docker:
+      - image: deliveroo/circleci:0.2.8
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: build
+          command: ci build
```

Comment:
why `ci build` instead of `make dockerbuild`?

Reply:
The `ci` command is a helper command provided by the base image `deliveroo/circleci:0.2.8` https://github.com/deliveroo/circleci#custom-helper-ci. It allows us to use the `docker-compose.ci.yml` file.